api/src/main/java/com/redhat/ipaas/api/v1/rest/util/PaginationOptions.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.api.v1.rest.util;
+
+public interface PaginationOptions {
+
+    int getPage();
```

Comment:
tbh Sonarqube helped me quite a bit with certain metrics. I.e. the circular package dependency was very helpful so that refactoring later was much easier (i.e moving things around). Also it detected some real bugs pro-actively. But tbh I stopped using it since some time as it really eats time and as you said, no one was really convinced about using it. ramp-up speed is king.

Sonarqube is important for single-person projects where you habe no human reviews. So I considered it always as my mechanical review friend ;-). Hence its probably no that important for us here.